types/yargs/index.d.ts

```diff
@@ -226,6 +227,7 @@ declare namespace yargs {
         skipValidation(key: string | ReadonlyArray<string>): Argv<T>;
 
         strict(): Argv<T>;
+        strict(enable: boolean): Argv<T>;
```

Comment:
It's actually `enabled` not `enable`.

Reply:
That's what I would use as well but there are other occurrences of `enable` in the file.

Reply:
What matters is the name of the parameter and it clear that it's `enabled`. https://github.com/yargs/yargs/blob/v12.0.5/yargs.js#L761

Reply:
I agree, but then all the others should be changed as well. Do you want that to go in this PR?

Reply:
I don't see any other case in `yargs` code where the parameter is `enabled` and we have it as `enable`, with the exception of `showHelpOnFail` that should be corrected. The others that you see are so because there is no specific parameter name when the passed parameter is a `boolean` (i.e. `env`, `version`),

If you look closely `exitProcess` also has an `enabled` parameter although the docs describe it as `enable`. We stick with the code.

Reply:
Sorry, I don't get the difference between `showHelpOnFail` (which should indeed be corrected) and `env` or `version`. The type is the same and they all toggle a feature. Since "enable" is an imperative, I would expect a variable called `enable` to be a function that enables something.